### DISCUSSION TITLE
Suggestion: Allow following people but only with certain labels/categories

### DISCUSSION THREAD

**Discussion by GreatBigWhiteWorld** [mastodon/mastodon#23260] [General]
Problem:
As the host of a mastodon user, we may want to post all kinds of stuff, some techs, some politics, some funny memes etc. But as followers of a mastodon user, we may be only interested in certain kinds of post from the one you are following.
Solution:
Users can optionally add multiple labels/categories to their account, from which can be chosen to label/categorize when they make a post. On the followers side, anyone who starts to follow a user can be given the choice to choose certain labels/categories set by the one he's following.

Of course, whether a user wants to add labels/categories to his/her account, or how he/she wants to make them, are totally by choice. 

I don't know about you, but having to read every kind of post from the one I follow is the major reason I'm not keen on most social media services. As a poster, I also feel that users can feel more free when they post random stuff.

**Comment by trwnh** [comment]
#1208

**Reply by GreatBigWhiteWorld** [reply]
Many thanks for the link.
